### PR TITLE
Payout 0.9.30

### PR DESCRIPTION
**Pull request by aie0**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- [C,D] Updated Substrate to polkadot-v0.9.30
+- [C,D] Updated Substrate to polkadot-v0.9.31
```
Comment (rakanalh) [inline CHANGELOG.md:47]: Undo this change.

File: pallets/ddc-customers/Cargo.toml [hunk 2]
```diff
@@ -17,7 +18,7 @@ rand_chacha = { version = "0.2", default-features = false, optional = true }
 scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
 
 # Cere
-ddc-primitives = { version = "0.1.0", default-features = false, path = "../../primitives" }
+ddc-primitives = { version = "4.8.2", default-features = false, path = "../../primitives" }
 ddc-traits = { version = "0.1.0", default-features = false, path = "../../traits" }
```
Comment (rakanalh) [inline pallets/ddc-customers/Cargo.toml:22]: This ddc-traits crate should have 4.8.2 version as well.

File: pallets/ddc-nodes/Cargo.toml [hunk 3]
```diff
@@ -5,7 +5,7 @@ edition = "2021"
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = "3.1.5", default-features = false, features = ["derive"] }
-ddc-primitives = { version = "0.1.0", default-features = false, path = "../../primitives" }
+ddc-primitives = { version = "4.8.2", default-features = false, path = "../../primitives" }
 ddc-traits = { version = "0.1.0", default-features = false, path = "../../traits" }
```
Comment (rakanalh) [inline pallets/ddc-nodes/Cargo.toml:9]: same here

File: pallets/ddc-payouts/Cargo.toml [hunk 4]
```diff
@@ -6,7 +6,7 @@ edition = "2021"
 [dependencies]
 byte-unit = { version = "4.0.19", default-features = false, features = ["u128"] }
 codec = { package = "parity-scale-codec", version = "3.1.5", default-features = false, features = ["derive"] }
-ddc-primitives = { version = "0.1.0", default-features = false, path = "../../primitives" }
+ddc-primitives = { version = "4.8.2", default-features = false, path = "../../primitives" }
 ddc-traits = { version = "0.1.0", default-features = false, path = "../../traits" }
```
Comment (rakanalh) [inline pallets/ddc-payouts/Cargo.toml:10]: same here

File: pallets/ddc-staking/Cargo.toml [hunk 5]
```diff
@@ -5,7 +5,7 @@ edition = "2021"
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = "3.1.5", default-features = false, features = ["derive"] }
-ddc-primitives = { version = "0.1.0", default-features = false, path = "../../primitives" }
+ddc-primitives = { version = "4.8.2", default-features = false, path = "../../primitives" }
 ddc-traits = { version = "0.1.0", default-features = false, path = "../../traits" }
```
Comment (rakanalh) [inline pallets/ddc-staking/Cargo.toml:9]: same here

**Review by rakanalh — CHANGES_REQUESTED** [review]

**Review by rakanalh — APPROVED** [review]

**Review by yahortsaryk — APPROVED** [review]
